### PR TITLE
HTTP Source

### PR DESCRIPTION
**Pull request by odacremolbap**
Add HTTP Source docs

### CODE REVIEW

File: docs/sources/http.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Event source for HTTP
```
Comment (daceynolan) [inline docs/sources/http.md:1]: Nitpicky -I hate to do this one haha.  To follow the MLA guidelines for titles,` source` here should be capitalized as `Source `

File: docs/sources/http.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+This event source exposes a generic HTTP endpoint that turns user requests into [CloudEvents][ce] to be consumed by other Triggermesh components. This source can fit integration scenarios where a webhook that might receive events from an external service is required.
+
+## Prerequisites
+
+An external client that executes HTTP requests.
+When using basic auth, a secret containing the password is required.
+
+## Create the HTTP source integration
```
Comment (daceynolan) [inline docs/sources/http.md:10]: Nitpicky -I hate to do this one haha. To follow the MLA guidelines for titles, `Create the HTTP source integration `here should be capitalized as` Create the HTTP Source Integration`

**Review by daceynolan — CHANGES_REQUESTED** [review]
I hate to be so nitpicky. We just changed all of the titles to follow the MLA guidelines yesterday :)

File: docs/sources/http.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# Event source for HTTP
+
+This event source exposes a generic HTTP endpoint that turns user requests into [CloudEvents][ce] to be consumed by other Triggermesh components. This source can fit integration scenarios where a webhook that might receive events from an external service is required.
```
Comment (daceynolan) [inline docs/sources/http.md:3]: `Triggermesh` here should be `TriggerMesh`

**Review by daceynolan — CHANGES_REQUESTED** [review]

File: docs/sources/http.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+## Prerequisites
+
+An external client that executes HTTP requests.
+When using basic auth, a secret containing the password is required.
+
+## Create the HTTP source integration
+
+Create an instance of the HTTP Source at Triggermesh as part of a Bridge.
```
Comment (daceynolan) [inline docs/sources/http.md:12]: Triggermesh here should be TriggerMesh

**Review by daceynolan — CHANGES_REQUESTED** [review]

**Comment by odacremolbap** [comment]
@daceynolan thanks
 https://github.com/triggermesh/docs/pull/76/commits/848b0c3e430f51fb5392ea974ef0922a696a333e

**Review by daceynolan — APPROVED** [review]
Thanks @odacremolbap

**Pull request merged**